guides/source/action_view_overview.md

```diff
@@ -472,38 +510,39 @@ Default values can also be provided:
 <%= message %>
 ```
 
-Rendering the partial without a `:message` local variable uses the provided default value:
+Rendering the partial without a `:message` local variable uses the default value set in the `locals:` magic comment:
 
 ```ruby
 render "messages/message"
 # => "Hello, world!"
 ```
 
-Rendering the partial with additional local variable arguments will raise an exception:
+Rendering the partial with local variables not specified in the `local:` magic comment will also raise an exception:
 
 ```ruby
-render "messages/message", unknown_local: "will raise"
-# => ActionView::Template::Error: missing local: :unknown_local for app/views/messages/_message.html.erb
+render "messages/message", greeting: "will raise"
+# => ActionView::Template::Error: missing local: :greeting for app/views/messages/_message.html.erb
 ```
 
-Optional local variable arguments can be splatted:
+You can allow optional local variable arguments with the double splat `**` operator:
 
 ```erb
+
 <%# app/views/messages/_message.html.erb %>
 
 <%# locals: (message: "Hello, world!", **attributes) -%>
 <%= tag.p(message, **attributes) %>
 ```
 
-Or `locals` can be disabled entirely:
+If you set the `locals:` to empty `()`, partial local variable are disabled entirely:
```

Comment:
To simplify, a suggestion: "Or you can disable `locals` entirely by setting the `locals:` to empty `()`:"